manifests/balancermember.pp

```diff
@@ -40,8 +40,8 @@
 #
 define apache::balancermember (
   String $balancer_cluster,
-  String $url = "http://${$facts['networking']['fqdn']}/",
-  Array $options = [],
+  Stdlib::HTTPUrl $url = "http://${$facts['networking']['fqdn']}/",
```

Comment:
Just for context, https://httpd.apache.org/docs/2.4/mod/mod_proxy.html#balancermember is the documentation. That just mentions url, but I think HTTP/HTTPS is sufficient in schemes.

Reply:
Just as an additional Note:  `stdlib/types/httpurl.pp`:  
```
type Stdlib::HTTPUrl = Pattern[/(?i:\Ahttps?:\/\/.*\z)/]
```
So, HTTPS is included there.

Reply:
Yes, I was mostly thinking out loud. Should have been explicit in that.